setup.py

```diff
@@ -21,14 +21,14 @@
 long_description = (cwd / "README.md").read_text()
 
 SPECIAL_DEPENDENCIES_FOR_RELEASE = [
-    "idaes-pse==2.0.*",  # from PyPI
+    "idaes-pse==2.1.*",  # from PyPI
 ]
 
 SPECIAL_DEPENDENCIES_FOR_PRERELEASE = [
     # update with a tag from the nawi-hub/idaes-pse
     # when a version of IDAES newer than the latest stable release from PyPI
     # will become needed for the watertap development
-    "idaes-pse==2.0.*",
+    "idaes-pse==2.0.0",
```

Comment:
@agarciadiego You can apply this change to use IDAES 2.1 in the CI runs:

```suggestion
    "idaes-pse==2.1.0rc0",
```